events/SignIn/PasswordRecoveryRequested .json

```diff
@@ -9,6 +9,23 @@
     "mitreAttackTechniques": [
         "T1078 - Valid Accounts"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1531 - Account Access Removal",
+            "reason": "An attacker might use the password recovery process to lock out legitimate users by changing the password, effectively removing their access."
+        },
+        {
+            "technique": "T1190 - Exploit Public-Facing Application",
+            "reason": "If there is a vulnerability in the password recovery mechanism of a public-facing application (such as the AWS console), an attacker might exploit this to gain unauthorized access."
```

Comment:
I agree, but the same can be said of all other APIs if they had a vulnerability an attacker could exploit them.

Reply:
Removed.